src/background/runtime/BackgroundRuntime.ts

```diff
@@ -6,22 +6,24 @@ import { LockService } from '@src/background/services/lock/LockService';
 import { OnboardingService } from '@src/background/services/onboarding/OnboardingService';
 import { BridgeService } from '@src/background/services/bridge/BridgeService';
 import ModuleManager from '../vmModules/ModuleManager';
+import { WalletService } from '../services/wallet/WalletService';
 
 @singleton()
 export class BackgroundRuntime {
   constructor(
     private connectionService: ConnectionService,
     private lockService: LockService,
     private onboardingService: OnboardingService,
-    private bridgeService: BridgeService
+    private bridgeService: BridgeService,
+    private walletService: WalletService
   ) {}
 
   activate() {
     this.onInstalled();
     this.registerInpageScript();
     this.addContextMenus();
 
-    ModuleManager.init();
+    ModuleManager.init(this.walletService);
```

Comment:
Can't import the `WalletService` inside the `ModuleManager` - I'm getting the `chicken-or-the-egg` problem.

I'd be happy to hear better suggestions 🤗

Reply:
> The WalletService does not require the ModuleManager.

No, but it requires `SecretsService`, which requires `AccountsService`, which requires `WalletService` again.

Anyhow, I managed to make it work in this commit: cfc896f4014be3c73bce25aec42e7ac134eb824b.

That being said, I think we should refactor the circular dependency listed above. I created a ticket:
* https://ava-labs.atlassian.net/browse/CP-9160